modules/web/server/src/main/scala/navigate/web/server/http4s/WebServerLauncher.scala

```diff
@@ -307,4 +314,16 @@ object WebServerLauncher extends IOApp with LogInitialization {
       }
       .void
 
+  def guiderQualityPoll(
+    eng: NavigateEngine[IO],
+    t:   Topic[IO, GuidersQualityValues]
+  ): Stream[IO, Unit] =
+    Stream
+      .fixedRate[IO](FiniteDuration(1, TimeUnit.SECONDS))
+      .evalMap(_ => eng.getGuidersQuality)
+      .evalMapAccumulate(none) { (acc, gs) =>
```

Comment:
So this emits an event to a fixed rate, not when Guiders Quality change

Reply:
The values change at the acquisition rate, that can be up to 200 Hz for OIWFS. So the fixed rate actually lowers it to human read level.